client/blocks/comment-detail/style.scss

```diff
@@ -18,6 +18,24 @@
 	}
 }
 
+.comment-detail__transition-enter {
+	opacity: 0.01;
```

Comment:
Fun fact: at some point the transitions weren't working and I couldn't figure out why. The only apparent difference with the docs was that I was using `0` and they `0.01`, so I just tried that.
Eventually I figured out what was my mistake (I was using `ReactCSSTransitionGroup` around the wrong component), but I never went back to that.

Can't seem to find out why they use `0.01`, but for some reason it seems to me that with `0.01` the transitions are a tiny little bit smoother (might very well be my own mind playing tricks on me though).